src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/IssueAttribute.cs

```diff
@@ -22,6 +23,9 @@ namespace Infrastructure.Common
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
     public class IssueAttribute : WcfSkippableAttribute
     {
+        private static bool s_includeAllIssues = DetectIssuesToInclude();
+        private static HashSet<int> s_includeIssuesHash;
```

Comment:
I hear you both. I simplified to a single static hashset with the semantics similar to what @zhenlan proposed.  But to minimize confusion, I avoid all use of "skip" or "ignore" in member names and only use "include".